diff -up rocminfo-rocm-6.3.0/CMakeLists.txt.1~ rocminfo-rocm-6.3.0/CMakeLists.txt
--- rocminfo-rocm-6.3.0/CMakeLists.txt.1~	2024-12-16 01:00:59.357709201 +0100
+++ rocminfo-rocm-6.3.0/CMakeLists.txt	2024-12-16 01:08:40.030451230 +0100
@@ -131,7 +131,7 @@ add_definitions(-DLITTLEENDIAN_CPU=1)
 #
 # Linux Compiler options
 #
-set(ROCMINFO_CXX_FLAGS -std=c++11)
+set(ROCMINFO_CXX_FLAGS $ENV{CXXFLAGS} -std=c++11)
 set(ROCMINFO_CXX_FLAGS ${ROCMINFO_CXX_FLAGS} -fexceptions)
 set(ROCMINFO_CXX_FLAGS ${ROCMINFO_CXX_FLAGS} -fno-rtti)
 set(ROCMINFO_CXX_FLAGS ${ROCMINFO_CXX_FLAGS} -fno-math-errno)
